core/forkid/forkid_test.go

```diff
@@ -55,8 +55,8 @@ func TestCreation(t *testing.T) {
 				{4369999, ID{Hash: checksumToBytes(0x3edd5b10), Next: 4370000}}, // Last Spurious block
 				{4370000, ID{Hash: checksumToBytes(0xa00bc324), Next: 7280000}}, // First Byzantium block
 				{7279999, ID{Hash: checksumToBytes(0xa00bc324), Next: 7280000}}, // Last Byzantium block
-				{7280000, ID{Hash: checksumToBytes(0x668db0af), Next: 0}},       // First and last Constantinople, first Petersburg block
-				{7987396, ID{Hash: checksumToBytes(0x668db0af), Next: 0}},       // Today Petersburg block
+				{7280000, ID{Hash: checksumToBytes(0x668db0af), Next: 9069000}}, // First and last Constantinople, first Petersburg block
+				{9069001, ID{Hash: checksumToBytes(0x879d6e30), Next: 0}},       // Today Istanbul block
```

Comment:
Please change the comment to `First Istanbul block`, and change the number to `9069000` from `9069001`.

Please add another clause:

```go
{10000000, ID{Hash: checksumToBytes(0x879d6e30), Next: 0}},       // Future Istanbul block
```